Arcs/Arcs.cpp

```diff
@@ -26,29 +30,39 @@ PROGRAM " " PACKAGE_VERSION "\n"
 
 static const char USAGE_MESSAGE[] =
 PROGRAM " " PACKAGE_VERSION "\n"
-"Usage: arcs [OPTION]... ALIGNMENTS...\n"
 "\n"
-"ALIGNMENTS may be a SAM or BAM file.\n"
-"The output of the aligner may be piped directly into ARCS by setting\n"
-"ALIGNMENTS to /dev/stdin, in which case it must be in SAM format.\n"
+"Usage: arcs [Options] <alignment file list>  or\n"
+"       arcs [Options] --arks -f <contig sequence file> <chrom file list>\n"
 "\n"
-"Paired reads must occur consecutively (interleaved) in the SAM/BAM file.\n"
-"The output of the aligner may either not be sorted,\n"
-"or may be sorted by read name using samtools sort -n.\n"
-"The SAM/BAM file must not be sorted by coordinate position.\n"
+"CHECKS FOR DEFAULT(ARCS) METHOD:\n"
+"       Aligment files are REQUIRED either in command or in fofFile. Pay attention to have only alignment files in fofFile.\n"
 "\n"
-"The barcode may be found in either the BX:Z:BARCODE SAM tag,\n"
-"or in the read (query) name following an underscore, READNAME_BARCODE.\n"
-"In the latter case the barcode must be compsed entirely of nucleotides.\n"
+"       Alignments may be a SAM or BAM file.\n"
+"       The output of the aligner may be piped directly into ARCS by setting\n"
+"       Alignments to /dev/stdin, in which case it must be in SAM format.\n"
 "\n"
-"The contig sequence lengths must either be present in the SAM header\n"
-"or provided by the -f option.\n"
+"       Paired reads must occur consecutively (interleaved) in the SAM/BAM file.\n"
+"       The output of the aligner may either not be sorted,\n"
+"       or may be sorted by read name using samtools sort -n.\n"
+"       The SAM/BAM file must not be sorted by coordinate position.\n"
 "\n"
-" Options:\n"
+"       The barcode may be found in either the BX:Z:BARCODE SAM tag,\n"
+"       or in the read (query) name following an underscore, READNAME_BARCODE.\n"
+"       In the latter case the barcode must be compsed entirely of nucleotides.\n"
 "\n"
-"   -f, --file=FILE       FASTA file of contig sequences to scaffold [optional]\n"
-"   -a, --fofName=FILE    text file listing input SAM/BAM filenames\n"
-"   -s, --seq_id=N        min sequence identity for read alignments [98]\n"
+"       The contig sequence lengths must either be present in the SAM header\n"
+"       or provided by the -f option.\n"
+"\n"
+"CHECKS FOR ARKS METHOD:\n"
+"       Contig seqeunces are REQUIRED by the -f option.\n"
+"       Chromium read files are REQUIRED either in command or in fofFile. Pay attention to have only read files in fofFile.\n"
+"\n"
+"       Contig sequences to further scaffold and can be in either FASTA or FASTQ format.\n"
```

Comment:
```suggestion
```

I know this was in the previous ARKS help message but I've never come across a case where contigs are in fastq file, so let's just take this out for now.